alpaca/common/models/trading.py

```diff
@@ -1,28 +1,174 @@
+from ..enums import AssetClass, AssetStatus, AssetExchange, OrderStatus
+from pydantic import Field
+
+
+class Position(BaseModel):
+    """
+    Attributes:
+        asset_id (UUID): ID of the asset.
+        symbol (str): Symbol of the asset.
+        exchange (str): Exchange name of the asset.
+        asset_class (str): Name of the asset's asset class.
+        avg_entry_price (str): The average entry price of the position.
+        qty (str): The number of shares of the position.
+        side (str): "long" or "short" representing the side of the position.
+        market_value (str): Total dollar amount of the position.
+        cost_basis (str): Total cost basis in dollars.
+        unrealized_pl (str): Unrealized profit/loss in dollars.
+        unrealized_plpc (str): Unrealized profit/loss percent.
+        unrealized_intraday_pl (str): Unrealized profit/loss in dollars for the day.
+        unrealized_intraday_plpc (str): Unrealized profit/loss percent for the day.
+        current_price (str): Current asset price per share.
+        lastday_price (str): Last day’s asset price per share based on the closing value of the last trading day.
+        change_today (str): Percent change from last day's price.
+    """
+
+    asset_id: UUID
+    symbol: str
+    exchange: str
+    asset_class: str
+    avg_entry_price: str
+    qty: str
+    side: str
+    market_value: str
+    cost_basis: str
+    unrealized_pl: str
+    unrealized_plpc: str
+    unrealized_intraday_pl: str
+    unrealized_intraday_plpc: str
+    current_price: str
+    lastday_price: str
+    change_today: str
+
+
+class ClosePositionResponse(BaseModel):
+    """
+    Attributes:
+        order_id (UUID): ID of order that was created to liquidate the position.
+        status_code (int): Status code corresponding to the request to liquidate the position.
+    """
+
+    order_id: UUID
+    status_code: int
+
+
+class Order(BaseModel):
+    """
+    Attributes:
+        id (UUID): Order ID generated by Alpaca.
+        client_order_id (UUID): Client unique order ID
+        created_at (datetime): Timestamp when the Order was created.
+        updated_at (datetime): Timestamp when the Order was last updated.
+        submitted_at (datetime): Timestamp when the Order was submitted.
+        filled_at (Optional[datetime]): Timestamp when the Order was filled.
+        expired_at (Optional[datetime]): Timestamp when the Order expired at.
+        canceled_at (Optional[datetime]): Timestamp when the Order was canceled.
+        failed_at (Optional[datetime]): Timestamp when the Order failed at.
+        replaced_at (Optional[datetime]): Timestamp when the Order was replaced by a new Order.
+        replaced_by (Optional[UUID]): ID of Order that replaces this Order.
+        replaces (Optional[UUID]): ID of Order which this Order replaces.
+        asset_id (UUID): ID of the asset.
+        symbol (str): Symbol of the asset.
+        asset_class (str): Asset class of the asset.

```

Comment:
`AssetClass`, `OrderClass`, `OrderType`, `OrderSide`, `TimeInForce` should preferably be enums.